.repo/sample-list-generator/src/libs/api-loader.ts

```diff
@@ -0,0 +1,20 @@
+import path from 'path';
+import fs from 'fs';
+import type { ExtensionApiMap } from '../types';
+import { isFileExistsSync } from '../utils/filesystem';
+
+export const loadExtensionApis = (): ExtensionApiMap => {
+  const filePath = path.join(__dirname, '../../extension-apis.json');
+
+  // check if extension-apis.json exists
+  if (!isFileExistsSync(filePath)) {
+    console.error(
+      'extension-apis.json does not exist. Please run "npm run prepare-chrome-types" first.'
+    );
+    process.exit(1);
+  }
+
+  let _EXTENSION_API_MAP = fs.readFileSync(filePath, 'utf8');
+  let EXTENSION_API_MAP = JSON.parse(_EXTENSION_API_MAP) as ExtensionApiMap;
+  return EXTENSION_API_MAP;
```

Comment:
Something like this probably works right? We don't need the `as` as far as I know because JSON.parse returns any, so it should be fine to just guess that from the function return signature.

```suggestion
  let data = fs.readFileSync(filePath, 'utf8');
  return JSON.parse(data);
```